rollup.config.js

```diff
@@ -7,7 +7,7 @@ import pkg from './package.json'
 
 const env = process.env.NODE_ENV
 
-const extensions = ['.js', '.ts', '.json']
+const extensions = ['.js', '.ts', '.tsx', '.json']
```

Comment:
I don't know rollup so I'm not sure if this is necessary, but it seemed reasonable